treeherder/model/models.py

```diff
@@ -1251,6 +1251,23 @@ def elastic_search_insert(self):
             return es_line
 
 
+class Group(models.Model):
+    """
+    The test harness Group.  Sometimes this is a Manifest, but not always.
+    Not to be confused with JobGroup which is Treeherder specific.
+    """
+    id = models.BigAutoField(primary_key=True)
+    name = models.CharField(max_length=200, unique=True)
```

Comment:
I thought the paths were normally paths relative to the mozilla-central (or whatever) source directory? If so, analysing what the deepest path is there might be helpful? (And adding a bit)

For any paths that are absolute paths, I think that's a bug that should be fixed in the harness.